iroha/src/torii.rs

```diff
@@ -1,20 +1,22 @@
-use crate::prelude::*;
+use crate::{peer::Message, prelude::*, MessageSender};
 use futures::{executor::ThreadPool, lock::Mutex};
 use iroha_derive::log;
 use iroha_network::prelude::*;
 use std::{convert::TryFrom, sync::Arc};
 
-const QUERY_REQUEST_HEADER: &str = "/queries";
-const COMMAND_REQUEST_HEADER: &str = "/commands";
+const QUERY_URI: &str = "/queries";
+const INSTRUCTIONS_URI: &str = "/commands";
+const BLOCKS_URI: &str = "/blocks";
```

Comment:
not clear name. Could be treated as either request blocks or send blocks

Reply:
@kamilsa - please, let's discuss entypoints naming in RFC https://wiki.hyperledger.org/display/iroha/Web+API